transformations/transformer_text_generation/transformation.py

```diff
@@ -0,0 +1,167 @@
+import logging
+import math
+
+import numpy as np
+import torch
+from transformers import GPT2LMHeadModel, pipeline
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+logging.basicConfig(
+    format="%(asctime)s - %(levelname)s - %(name)s -   %(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+    level=logging.INFO,
+)
+logger = logging.getLogger(__name__)
+
+
+def set_seed(seed: int, no_cuda: bool):
+    np.random.seed(seed)
+    torch.manual_seed(seed)
+    if not no_cuda:
+        torch.cuda.manual_seed_all(seed)
+
+
+class TransformerTextGeneration(SentenceOperation):
```

Comment:
I think this transformation should be a heavy transformation as we are using a transformer model. If that's the case could you please make 'heavy = True'